datalake/backend/endpoints/dataset_versions/create.py

```diff
@@ -0,0 +1,106 @@
+"""Import dataset-version function."""
```

Comment:
This docstring doesn't add too much value to me. I would change it to something more descriptive (like "Dataset versions handler function") or remove it.